src/Sylius/Bundle/AdminBundle/spec/Action/Account/RequestPasswordResetActionSpec.php

```diff
@@ -59,7 +59,7 @@ public function it_sends_reset_password_request_to_message_bus(
             ->willReturn($form)
         ;
 
-        $form->handleRequest($request)->shouldBeCalled();
+        $form->handleRequest($request)->willReturn($form)->shouldBeCalled();
```

Comment:
`$form->handleRequest($request)->willReturn($form)` is enough, isn't it?

Reply:
`->shouldBeCalled()` does not break anything but indeed does not have to be here 🖖